common/src/main/java/org/opensearch/ml/common/model/Guardrails.java

```diff
@@ -0,0 +1,120 @@
+package org.opensearch.ml.common.model;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import org.opensearch.core.xcontent.ToXContentObject;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.core.xcontent.XContentParser;
+
+import java.io.IOException;
+
+import static org.opensearch.core.xcontent.XContentParserUtils.ensureExpectedToken;
+
+@EqualsAndHashCode
+@Getter
+public class Guardrails implements ToXContentObject {
+    public static final String TYPE_FIELD = "type";
+    public static final String ENGLISH_DETECTION_ENABLED_FIELD = "english_detection_enabled";
```

Comment:
Do we have detect english function in this PR ? If no, let's not add this part for now, that add confusion if someone read this code.

Reply:
I just reserved it, as we will support in next releases I think. Anyway I can remove it.

Reply:
I don't want to introduce another version control in next release, so prefer to keep it.

Reply:
I would prefer to remove it and once we support it, it would make sense this to be language detection, so can configure different language depending on what their model is tuned for.

Reply:
removed in another pr.